app/jobs/channels/approve_channel_transfer_job.rb

```diff
@@ -1,41 +1,37 @@
 module Channels
-  class ApproveChannelTransfer < BaseService
```

Comment:
Would you add `sidekiq_options retry: X` here or to `application_job.rb`? I don't think there isn't a default set. The `max_retries` in sidekiq.yml seems to be fooling us: https://github.com/mperham/sidekiq/wiki/Active-Job#customizing-error-handling

Reply:
Based off the documentation It seems like ActiveJob will automatically retry and then kick it over to sidekiq? 

> The default AJ retry scheme is 3 retries, 5 seconds apart. Once this is done (after 15-30 seconds), AJ will kick the job back to Sidekiq, where Sidekiq's retries with exponential backoff will take over.

Where then Sidekiq will retry up to `max_retries`